lib/opentok/token_generator.rb

```diff
@@ -81,6 +81,15 @@ def generate_token
         end
         data_params[:connection_data] = data
       end
+
+      if opts[:initial_layout_classes]
```

Comment:
@Lucashuang0802 Good point! I was just merging in the old branch, these parameters must have changed. Thanks for noticing this, I'll fix it.